src/webhooks/discord/discord.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { Prisma } from '@prisma/client';
+import { WebhookClient } from 'discord.js';
+import config from '../../configs/config';
+import { CREATOR_REGISTERED } from './templates/creatorRegistered';
+import { MessagePayload } from 'discord.js';
+import { CREATOR_FILES_UPDATED } from './templates/creatorFilesUpdated';
+import { CreatorFiles } from './dto/types';
+import { CREATOR_VERIFIED } from './templates/creatorVerified';
+
+@Injectable()
+export class DiscordService {
+  private readonly apiUrl: string;
+  private readonly discord: WebhookClient;
+  private readonly payload: MessagePayload;
+
+  constructor() {
+    this.discord = new WebhookClient({
+      id: process.env.DISCORD_CLIENT_ID,
+      token: process.env.DISCORD_WEBHOOK_TOKEN,
+    });
+    this.apiUrl = config().client.dPublisherUrl;
+    this.payload = new MessagePayload(
+      this.discord.client,
+      this.discord.options,
+    );
+  }
+  async notifyCreatorRegistration(creator: Prisma.CreatorCreateInput) {
```

Comment:
lets put in an empty line above this function and my eyes will be thankful to you 🙏